ground/src/main/java/com/google/android/ground/ui/datacollection/DataCollectionViewModel.kt

```diff
@@ -15,10 +15,57 @@
  */
 package com.google.android.ground.ui.datacollection
 
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.LiveDataReactiveStreams
+import com.google.android.ground.model.submission.Submission
+import com.google.android.ground.repository.SubmissionRepository
+import com.google.android.ground.rx.Loadable
+import com.google.android.ground.rx.annotations.Hot
 import com.google.android.ground.ui.common.AbstractViewModel
+import com.google.android.ground.ui.common.LocationOfInterestHelper
+import io.reactivex.Flowable
+import io.reactivex.processors.BehaviorProcessor
+import io.reactivex.processors.FlowableProcessor
 import javax.inject.Inject
 
 /**
  * View model for the Data Collection fragment.
  */
-class DataCollectionViewModel @Inject internal constructor() : AbstractViewModel()
\ No newline at end of file
+class DataCollectionViewModel @Inject internal constructor(
+    private val submissionRepository: SubmissionRepository,
+    private val locationOfInterestHelper: LocationOfInterestHelper
+) : AbstractViewModel() {
+    val submission: @Hot(replays = true) LiveData<Loadable<Submission>>
+    val title: @Hot(replays = true) LiveData<String>
```

Comment:
I think we can model these more specifically as `jobName` and `loiName` or `locationOfInterestName`. [I think abbreviations are fine as variable names]

Reply:
Done